app/forms/submissions/base_validator.rb

```diff
@@ -33,6 +34,10 @@ def initialize(trainee:)
       @trainee = trainee
     end
 
+    def requires_placements?
+      trainee.requires_placements? && FeatureService.enabled?(:trainee_placement)
```

Comment:
Feels like we should just put this feature flag check directly in `trainee.requires_placements?`?

Reply:
I was thinking that we will be like removing the flags after release, maybe we defer til then?

Reply:
My take on this is optimise for ease of removal. Fewer places we have to change the better.

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/pull/3734